components/mixins/experience-event/industry-verticals/experienceevent-travel-preferences.schema.json

```diff
@@ -0,0 +1,208 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/classes/travel-preferences",
```

Comment:
This is a mixin so , please fix the Schema ID to
"https://ns.adobe.com/xdm/mixins/travel-preferences"

Reply:
yes, this is a mixin.